plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Pom.java

```diff
@@ -65,6 +65,15 @@
 import org.xml.sax.SAXException;
 
 class Pom {
+  private static final String NAMESPACE_URI = "http://maven.apache.org/POM/4.0.0";//$NON-NLS-1$
+  private static final String DEPENDENCY_MANAGEMENT = "dependencyManagement"; //$NON-NLS-1$
```

Comment:
I've seen it done, but I wouldn't define constants for element and attribute names. The constants are no clearer than the string literals, arguably less so.

Reply:
done